anthos-bm-gcp-terraform/test/unit/main_test.go

```diff
@@ -1110,11 +1155,17 @@ func validateInstanceTemplateModule(goTester *testing.T, module *util.TFModule,
 					fmt.Sprintf("Invalid value for resources[%d].values.service_account[0].scopes[%d] in the instance_template child module", idx, it),
 				)
 			}
+
+			// assert.Nil(
+			// 	goTester,
+			// 	resource.Values.Gpu,
+			// 	fmt.Sprintf("Invalid value for GPU settings. Should be null by default."),
+			// )
```

Comment:
Is there a reason this chunk is commented out?

Reply:
This was a mistake. I have removed it in https://github.com/GoogleCloudPlatform/anthos-samples/pull/68/commits/bddfb4c3c8e7f075d95670e3fa24475e328c5fe6

Reply:
Oh, nvm, this has been addressed. :)